WcaOnRails/app/models/competition.rb

```diff
@@ -43,6 +43,12 @@ def must_have_at_least_one_event
       errors.add(:eventSpecs, "Competition must have at least one event")
     end
   end
+  validate :must_have_at_least_one_delegate, if: :isConfirmed?
+  def must_have_at_least_one_delegate
+    if delegate_ids.length == 0
+      errors.add(:delegates, "Competition must have at least one WCA delegate")
```

Comment:
It's weird that you check `delegate_ids`, but then add an error to the `delegates` field. If you change this to `delegate_ids` in both places, do you get a nice red box when there's an error?